packages/react/src/themes/teams-high-contrast/components/Checkbox/checkboxVariables.ts

```diff
@@ -0,0 +1,15 @@
+import { CheckboxVariables } from '../../../teams/components/Checkbox/checkboxVariables'
+
+export default (siteVars: any): Partial<CheckboxVariables> => {
+  return {
+    checkboxBorderColorHover: siteVars.accessibleYellow,
+    checkboxCheckedBackground: siteVars.accessibleCyan,
+    checkboxCheckedBackgroundHover: siteVars.accessibleYellow,
+    checkboxCheckedIndicatorColor: siteVars.colors.black,
+
+    toggleIndicatorColor: siteVars.colors.white,
+
+    disabledCheckboxCheckedIndicatorColor: siteVars.colors.black,
+    disabledToggleIndicatorColor: siteVars.colors.accessibleGreen,
```

Comment:
i don't think you need the `disabledToggleIndicatorColor` variable defined here since `foregroundDisabled1` already sets this to green in high contrast theme